src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+      case 'removeProperties':
+        action.keysToRemove.forEach(key => delete state[key]);
+        break;
+
+      case 'clear':
+        Object.keys(state).forEach(key => delete state[key]);
+        break;
+
+      default:
+        return console.error('Error, probably the wrong parameter was specified.');
```

Comment:
Either return something or log it
```suggestion
        console.error('Error, probably the wrong parameter was specified.');
```

Reply:
lint doesn't allow console.error so I returned a regular string